### ISSUE TITLE
Explanation On Giving Mod Armor Enchantments

### ISSUE THREAD

**Issue by SirR3AL** [fnar/minecraft-roguelike#57]
Hello,

I am developing a modpack and I am using your mod to give players the option to loot armors, ingots, weapons, and other items in the overworld rather than straight up crafting them. I have successfully gotten the armors and items to spawn in their respective chest but would to see how it is possible to have these armors spawn with random enchantments like the vanilla items as the enchantment side of your mod seems to be configured more for armor based on the vanilla materials and not modded ones (thermal expansion Platinum set and Thaumcraft Thaumium set). Your mod may perhaps is already coded to deal with these and I am just misinterpreting your wiki so that's why I wanted a little more information on how to configure it for this purpose. Thank you in advance!

**Comment by fnar** [comment]
I don't think this mod currently supports "naturally" enchanting modded materials and metals, unfortunately. This is due to the hardcoded nature of the RLD loot generation engine.

It could be theoretically possible in the future, but supporting naturally enchanting modded materials is not currently a priority, sadly, due to competition with bringing the mod to 1.14+.

I'll leave this issue here and open as a reminder for that feature request.

**Comment by fnar** [comment]
If you felt up for it, you could manually supply the items with their enchantments provided as NBT tags within RLD loot table configurations. This sounds insanely tedious, so I recommend against it, but's possible.

**Comment by fnar** [comment]
It would not be a far stretch to modify the current way that "weapons" or other "[equipment](https://github.com/fnar/minecraft-roguelike/wiki/Equipment-Loot-Items)" are specified in configuration such that it accepts an item name and enchants it like the default RLD weapons/equipment.

I'll continue to leave this open as an enhancement.
